src/main/editDelete/__tests__/preProcess.test.ts

```diff
@@ -0,0 +1,102 @@
+import preProcessTranscript from '../preProcess';
+
+describe('Test pre-processing JSON transcript into regular transcript', () => {
+  it('should produce expected transcript when given a JSON transcript', () => {
+    const jsonTranscript = {
+      confidence: 1,
+      words: [
+        {
+          word: 'abc',
+          start_time: 0,
+          duration: 1,
+        },
+        {
+          word: 'def',
+          start_time: 1,
+          duration: 1,
+        },
+      ],
+    };
+    const duration = 2;
+
+    const outputTranscript = preProcessTranscript(jsonTranscript, duration);
+
+    expect(outputTranscript).toEqual({
+      confidence: 1,
+      words: [
+        {
+          deleted: false,
+          duration: 1,
+          fileName: 'PLACEHOLDER FILENAME',
+          key: '0',
+          startTime: 0,
+          word: 'abc',
+        },
+        {
+          deleted: false,
+          duration: 1,
+          fileName: 'PLACEHOLDER FILENAME',
+          key: '1',
+          startTime: 1,
+          word: 'def',
+        },
+      ],
+    });
+  });
+
+  it('should fill blank spaces between words, and between last word and end', () => {
+    const jsonTranscript = {
+      confidence: -7,
+      words: [
+        {
+          word: 'heat',
+          start_time: 5,
+          duration: 2,
+        },
+        {
+          word: 'from',
+          start_time: 8,
+          duration: 3,
+        },
+        {
+          word: 'fire',
+          start_time: 11.2,
+          duration: 0.9,
+        },
+      ],
+    };
+    const duration = 15.77;
+
+    const outputTranscript = preProcessTranscript(jsonTranscript, duration);
+
+    expect(outputTranscript).toEqual({
```

Comment:
I'm surprised these test cases passed. I believe they need to be updated to included the new keys in `Word`

Reply:
I'm just now realizing you didn't create these and its just from Patrick's refactor that hasn't been merged with develop...